lib/fetch/index.js

```diff
@@ -1042,9 +1042,10 @@ async function httpFetch (fetchParams) {
       // Set response to a network error.
       response = makeNetworkError()
     } else if (request.redirect === 'manual') {
-      // Set response to an opaque-redirect filtered response whose internal
-      // response is actualResponse.
-      response = filterResponse(actualResponse, 'opaqueredirect')
+      // On the web this would return an `opaqueredirect` response, but
+      // those don't make sense server side.
+      // See https://github.com/nodejs/undici/issues/1193.
+      response = actualResponse
```

Comment:
```suggestion
      // Set response to an opaque-redirect filtered response whose internal
      // response is actualResponse.
      // NOTE(spec): On the web this would return an `opaqueredirect` response, but
      // those don't make sense server side.
      // See https://github.com/nodejs/undici/issues/1193.
      response = actualResponse
```

Reply:
thanks, done!